spec/project_config_spec.rb

```diff
@@ -59,6 +59,21 @@
         'test_event_not_running' => config_body['events'][3]
       }
 
+      expected_experiment_feature_map = {
+        '133331' => [
```

Comment:
An experiment having 2+ feature flags associated with it is an unexpected state. This points to me that the data we are testing with is incorrect / unexpected. Experiment to feature relationship is a many to one mapping and not many to many mapping.

cc @mikeng13

Reply:
I just checked and these are mutex experiments, for which we do allow multiple features to be associated. So I think this data representation is correct.

Reply:
Just synced offline with @aliabbasrizvi and turns out I had it backwards. An experiment can only be associated to one feature at a time. We should definitely fix this test data as it would otherwise suggest an erroneous case.